test/e2e/testing-manifests/statefulset/multivolumezk/statefulset.yaml

```diff
@@ -0,0 +1,128 @@
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name: multivolumezk
+spec:
+  serviceName: "multivolumezk"
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: multivolumezk
+      annotations:
+        pod.alpha.kubernetes.io/init-containers: '[
```

Comment:
Switch this one here, the rest of the examples can be handled as part of that issue (I have already seen PRs switching to fields).